giraffe/src/utils/formatters.ts

```diff
@@ -153,7 +161,11 @@ export const timeFormatter = ({
         }
         return String(Number(lhour))
       }
-      return lhour
+      if (is24hourLocale || UTC_TIME_ZONE.test(timeZone) || hour12 === false) {
```

Comment:
What state of the system do all these conditions being a certain value represent?

```ts
is24hourLocale || UTC_TIME_ZONE.test(timeZone) || hour12 === false
```

in other words, can you store the output of the above expression into a variable that names / describes the situation more eloquently than the three `or` ed expressions?

could you name it something like 

`const isUTCAnd24Hours = is24hourLocale || UTC_TIME_ZONE.test(timeZone) || hour12 === false`

or similar to better clue in what's happening?